src/spdx_tools/spdx/model/relationship.py

```diff
@@ -73,3 +73,6 @@ def __init__(
         comment: Optional[str] = None,
     ):
         check_types_and_set_values(self, locals())
+
+    def __hash__(self):
```

Comment:
From [python](https://docs.python.org/3.5/reference/datamodel.html#object.__hash__) docs:

If a class does not define an [__eq__()](https://docs.python.org/3.5/reference/datamodel.html#object.__eq__) method it should not define a [__hash__()](https://docs.python.org/3.5/reference/datamodel.html#object.__hash__) operation either;

(Note I am not a python expert.)